components/pkg-export-docker/src/docker.rs

```diff
@@ -53,17 +53,20 @@ pub struct DockerBuilder<'a> {
     name: String,
     /// A list of tags for the image.
     tags: Vec<String>,
+    /// Optional memory limit to pass to pass to the docker build
+    memory: Option<&'a str>,
```

Comment:
We are basically just passing the string on to `docker build`. We could be more explicit in our clapp docs that that is what we are soing and then let docker handle the validationi since it will error if the value is invalid.